zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -781,6 +784,11 @@ private static void sendToKafka(Collection collection) throws IOException {
         sendMessage (collection, reviewedUris, "Reviewed-uris");
     }
 
+    // valid uris for published CMD versions of a dataset (edition) - /dataset/{datatsetId}/editions/{edition}/versions/{version}/metadata
+    protected static boolean isValidUris (String uri){
```

Comment:
I'd rename this function as it's too broad. All the uris (or at least most) are presumably valid for something. The method name should indicate exactly what's being worked out. Eg. `isValidCMDDatasetURI` or something like that.